src/CSS/ExperimentTwo/ModalStyle.css

```diff
@@ -0,0 +1,31 @@
+.CloseLayout {
+  font-family: sans-serif;
+  text-align: center;
+}
+
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.6);
+}
+
+.modal-main {
+  position: fixed;
+  background: white;
+  width: 80%;
+  height: auto;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.display-block {
+  display: block !important;
```

Comment:
using `!important` is usually bad practice, do you really need it here?

Reply:
Yeah sure. Tbh I took the CSS from somewhere on internet and so didn't look into it careufully. I will be careful next time